### PR TITLE
chore(blog): add a post about kvrocks and go-redis

### PR DESCRIPTION
**Pull request by vmihailenco**
This post explains how to get started with Apache Kvrocks using go-redis client. It also
demonstrates how you can use OpenTelemetry and Uptrace to monitor Kvrocks on both client
and server sides.

Let me know if this is appropriate for Kvrocks blog and if any changes are needed.

### CODE REVIEW

**Comment by tisonkun** [comment]
Cool! Thank you @vmihailenco :)

I'll give it a review in days.

**Comment by git-hulk** [comment]
Many thanks to @vmihailenco

**Review by git-hulk — APPROVED** [review]
The format and content look good!

File: blog/2022-11-20-go-redis-kvrocks-opentelemetry/index.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+The metric looks like this in Uptrace:
+
+![Redis Metrics](used-disk-percent.png)
+
+See [OpenTelemetry Go Metrics API](https://uptrace.dev/opentelemetry/go-metrics.html) for details.
+
+## Useful links
+
+- [Kvrocks Server Installation](https://kvrocks.apache.org/docs/server-installation)
```
Comment (tisonkun) [inline blog/2022-11-20-go-redis-kvrocks-opentelemetry/index.md:246]:
```suggestion
- [Kvrocks Server Installation](/docs/server-installation)
```

**Review by tisonkun — CHANGES_REQUESTED** [review]
Thanks for your good blog! Comments inline.

File: blog/2022-11-20-go-redis-kvrocks-opentelemetry/index.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+
+Otel allows developers to collect and export telemetry data in a vendor agnostic way. With
+OpenTelemetry, you can instrument your application once and then add or change vendors without
+changing the instrumentation, for example, here is a list
+[DataDog competitors](https://uptrace.dev/get/compare/datadog-competitors.html) that support
+OpenTelemetry.
+
+## What is Uptrace?
+
+Uptrace is an [open-source APM tool](https://uptrace.dev/get/open-source-apm.html) that supports
```
[the remark this answers is not in the document]
Comment (vmihailenco) [reply blog/2022-11-20-go-redis-kvrocks-opentelemetry/index.md:134]: It is for SEO not marketing but fair point.

**Comment by vmihailenco** [comment]
@tisonkun applied all suggestions - thanks for the review :+1: PTAL

**Review by PragmaTwice — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]
Merging...

Thanks again for your post @vmihailenco!

**Comment by vmihailenco** [comment]
My pleasure! Thank you!

**Pull request merged**